### PR TITLE
Add Bittercup Companion - Redux

### PR DESCRIPTION
**Pull request by Superfly-Johnson**
I think a 

##### NPCs - Followers ##### subsection or something would be nice.

### CODE REVIEW

**Comment by sibir-ine** [comment]
That would be the `Encounters - Allies` section.

**Comment by Superfly-Johnson** [comment]
Fair enough, I'll update the commit very soon

File: masterlist.yaml [hunk 1]
```diff
@@ -1539,6 +1539,25 @@ plugins:
         util: '[FNVEdit v4.0.4b](https://www.nexusmods.com/newvegas/mods/34703)'
         nav: 5
 
+  - name: 'BittercupRedux.esp'
+    url:
+      - link: 'https://www.nexusmods.com/newvegas/mods/80781'
```
Comment (pStyl3) [inline masterlist.yaml]: Trailing slash `/` is missing.

**Review by pStyl3 — CHANGES_REQUESTED** [review]

**Review by pStyl3 — APPROVED** [review]

**Comment by pStyl3** [comment]
Thanks!

**Pull request merged**